sklearn/utils/__init__.py

```diff
@@ -21,6 +21,7 @@
 from ..utils.fixes import _Sequence as Sequence
 from .deprecation import deprecated
 from .. import get_config
+from ._print_versions import show_versions
```

Comment:
I think we wanted to import this in `sklearn/__init__.py` instead?

Reply:
+1